resources/lib/utils.py

```diff
@@ -149,5 +149,5 @@ def localize(string_id):
 
 def localize_time(time):
     """Localize time format"""
-    time_format = getRegion('time').replace(':%S', '')  # Strip off seconds
+    time_format = getRegion('time').replace('%H%H', '%H').replace(':%S', '')  # Strip off seconds
```

Comment:
I am concerned.
In what situation would that ever be %H%H where this would not be a problem otherwise?

I need to understand this before considering any solutions. It might as well be a bug upstream.